modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/upgrade-catch-all-check/LPD_27178.testjava

```diff
@@ -0,0 +1,20 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+/**
+ * @author Micaelle Silva
+ */
+public class LPD_27178 {
+
+	public void method() {
```

Comment:
@MicaelleSilvaa Declare the article and the folder as a parameter, please.

`method(Folder folder, JournalArticle journalArticle) `